tinkerbell/stack/templates/nginx.yaml

```diff
@@ -115,6 +117,7 @@ spec:
           path: {{ .Values.stack.hook.downloadsDest }}
           type: DirectoryOrCreate
       {{- end }}
+      {{- if .Values.stack.relay.enabled }}
```

Comment:
This could be combined with the proceeding line. https://helm.sh/docs/chart_template_guide/function_list/#and